usaspending_api/spending_explorer/tests/integration/test_spending_explorer.py

```diff
@@ -137,23 +137,23 @@ def test_unreported_data_actual_value_file_c(client):
             "model": FinancialAccountsByAwards,
             "financial_accounts_by_awards_id": -1,
             "submission_id": -1,
-            "award__recipient__recipient_name": "random_recipient_name_1",
+            "award__latest_transaction__assistance_data__awardee_or_recipient_legal": "random_recipient_name_1",
             "treasury_account_id": -1,
             "transaction_obligated_amount": -5,
         },
         {
             "model": FinancialAccountsByAwards,
             "financial_accounts_by_awards_id": -2,
             "submission_id": -1,
-            "award__recipient__recipient_name": "random_recipient_name_2",
+            "award__latest_transaction__assistance_data__awardee_or_recipient_legal": "random_recipient_name_2",
             "treasury_account_id": -1,
             "transaction_obligated_amount": -10,
         },
         {
             "model": FinancialAccountsByAwards,
             "financial_accounts_by_awards_id": -3,
             "submission_id": -1,
-            "award__recipient__recipient_name": "random_recipient_name_1",
+            "award__latest_transaction__assistance_data__awardee_or_recipient_legal": "random_recipient_name_1",
```

Comment:
All three are only assistance awards. Would be prudent to also test a procurement award returns the correct value

Reply:
Thanks for adding the additional records. Unfortunately the test cases aren't affected by them.